packages/cxl-ui/src/components/cxl-progress-bar.js

```diff
@@ -0,0 +1,18 @@
+import { customElement } from 'lit-element';
+import '@conversionxl/cxl-lumo-styles';
+import { ProgressBarElement } from '@vaadin/vaadin-progress-bar/src/vaadin-progress-bar';
+
+@customElement('cxl-playbook-progress-bar')
+export class CXLPlaybookProgressBar extends ProgressBarElement {
+  ready() {
+    super.ready();
+
+    document.addEventListener('cxl-playbook-accordion-changed', async (event) => {
```

Comment:
Possible to narrow down listening scope, ideally to specific accordion ID?

Reply:
Updated, please take a look now.